tests/jballerina-unit-test/src/test/resources/test-src/types/intersection/test_intersection_type_negative.bal

```diff
@@ -43,3 +43,7 @@ type Z record {
     X x;
     Y y;
 };
+
+type READONLY_REF1 readonly;
+type ImmutableTypeRef future<int>;
```

Comment:
Name isn't accurate?

Reply:
updated in [commit](https://github.com/ballerina-platform/ballerina-lang/pull/42025/commits/5561bfc4b63f1cc862efd176700b6eba79cbbbbd).